support.sh

```diff
@@ -4,11 +4,11 @@
 DOCKER="${DOCKER:-docker}"
 NSENTER="${NSENTER:-nsenter}"
 BRIDGE="${BRIDGE:-bridge}"
-BRCTL="${BRCTL:-brctl}"
 IPTABLES="${IPTABLES:-iptables}"
+IPVSADM="${IPVSADM:-ipvsadm}"
```

Comment:
do you want to add a preliminary function that checks for tool available before using them?
command -v ipvsadm
ipvsAvailable=$?

and then if not available skip the usage and throw a warning?

Reply:
@fcrisciani we check for them a little father down, but we exit if we don't find the right tools - ipvsadm is probably the only one that wouldn't be included by default with most distros.  On the one hand, it would be really nice to guarantee we can get ipvs state; but it would also be really unfortunate to run a support dump and have all this information missing because the script dies right at the beginning.

+1 to skip and throw warning